lxml-stubs/etree.pyi

```diff
@@ -32,7 +34,11 @@ def __getattr__(name: str) -> Any: ...
 # unnecessary constraint. It seems reasonable to constrain each
 # List/Dict argument to use one type consistently, though, and it is
 # necessary in order to keep these brief.
-_AnyStr = Union[str, bytes]
+if sys.version_info[0] >= 3:
+    _StrResult = str
+else:
+    _StrResult = Union[str, bytes]
+
```

Comment:
Python 3 will return unicode while Python 2 may return a unicode string or bytes.

Reply:
It has worked in practice with mypy and version checks appear in typeshed in a number of places (https://github.com/python/typeshed/blob/master/stdlib/builtins.pyi#L718)